docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+title: Multiple Browser Windows
+permalink: /documentation/guides/advanced-guides/multiple-browser-windows.html
+---
+# Multiple Browser Windows
+
+The TestCafe API includes methods that open, close, and switch between browser windows. You can test websites with pop-up windows and OAuth login forms, debug complex multi-window applications, or run multiple instances of the same web app side by side.
+
+>This is a beta feature. Browser support is limited to local instances of Chrome and Firefox. Videos and screenshots of child windows cannot be captured. The available functionality is subject to further revisions. Use at your own risk.
+
+## Handle client-side window events
+
+When your page launches a new window, the test automatically continues in the newly open window. When that window is closed, the test switches back to its parent.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Login page`
+    .page('https://login.wrike.com/login/');
+
+const googleButton = Selector('div.login-panel-footer__login-with > button');
+
+test('Login via Google', async t => {
+    await t
+        .click(googleButton)
+        .typeText('input[type=email]', 'This text will be entered inside the login dialog');
+});
+```
+
+## Open a new window
+
+Use the [t.openWindow](../../reference/test-api/testcontroller/openwindow.md) method to open a new window:
+
+```js
+import { Selector, ClientFunction } from 'testcafe';
+
+fixture `Test page`
+    .page('https://devexpress.github.io/testcafe/example/');
+
+test('Open a new window', async t => {
+    await t.openWindow('http://example.com');
+
+    const url = await t.eval(() => document.documentURI);
+    await t.expect(url).eql('http://example.com/');
+});
+```
+
+>All open browser windows share the same cookies and user storage. Support for per-window user sessions is not yet available.
```

Comment:
First, 'not yet' looks like a promise. Second (which I didn't say the last time), the documentation usually doesn't name features that aren't yet available. This limitation could be rephrased without mentioning that something specific is not yet here. For example:

*All open browser windows share the same cookies, user storage, and, therefore, the same user session.*